frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -175,68 +240,78 @@ function MissionDefinitionEditButtons({ missionDefinition, updateMissionDefiniti
-                                />
-                                <TextField
-                                    id="commentEdit"
-                                    multiline
-                                    rows={3}
-                                    label={TranslateText('Comment')}
-                                    value={form.comment ?? ''}
-                                    onChange={(changes: ChangeEvent<HTMLInputElement>) =>
-                                        setForm({ ...form, comment: changes.target.value })
-                                    }
-                                />
-                                <div>
-                                    <TextField
-                                        id="compact-textfield"
-                                        label={TranslateText('Days between inspections')}
-                                        unit={TranslateText('days')}
-                                        value={inspectionFrequencyDays}
-                                        onChange={(changes: ChangeEvent<HTMLInputElement>) =>
-                                            updateInspectionFrequencyFormDays(changes.target.value)
-                                        }
-                                    />
-                                    <TextField
-                                        id="compact-textfield"
-                                        label={TranslateText('Hours between inspections')}
-                                        unit={TranslateText('hours')}
-                                        value={inspectionFrequencyHours}
-                                        onChange={(changes: ChangeEvent<HTMLInputElement>) =>
-                                            updateInspectionFrequencyFormHours(changes.target.value)
-                                        }
-                                    />
-                                </div>
-                            </StyledFormSection>
-                            <StyledButtonSection>
-                                <Button onClick={handleSubmit} variant="outlined" color="primary">
-                                    {' '}
-                                    {TranslateText('Update mission definition')}{' '}
-                                </Button>
-                                <Button onClick={() => setIsEditDialogOpen(false)} variant="outlined" color="primary">
-                                    {' '}
-                                    {TranslateText('Cancel')}{' '}
-                                </Button>
-                            </StyledButtonSection>
-                        </StyledAutoComplete>
-                    </StyledDialog>
-                </StyledFormDialog>
-            )}
-        </>
+        <StyledDialog open={true}>
+            <StyledFormCard>
+                <Typography variant="h2">{TranslateText('Edit mission definition')}</Typography>
```

Comment:
Don't think we should use the word mission definition in the user interface. How about having "Edit name", "Edit comment" .. and such add header instead? And changing the text on the confirm button to just "Update" instead of "update mission definition"